backend/chatbot/chat_router.py

```
@@ -9,10 +9,17 @@
 from dotenv import load_dotenv
 from tenacity import retry, stop_after_attempt, wait_fixed
 from datetime import datetime
+from typing import AsyncGenerator
+
+# LangChain Imports
 from langchain_openai import ChatOpenAI
 from langchain_core.prompts import ChatPromptTemplate, SystemMessagePromptTemplate, HumanMessagePromptTemplate
 from langchain_core.output_parsers import StrOutputParser
-from typing import AsyncGenerator
+from langchain_core.chat_history import BaseChatMessageHistory
+from langchain_core.messages import AIMessage, HumanMessage
```

Ruff (0.11.9)

`langchain_core.messages.AIMessage` imported but unused

Remove unused import

(F401)